apps/src/tutorialExplorer/tutorialDetail.jsx

```diff
@@ -0,0 +1,168 @@
+/* A pop-up modal displaying information about a single tutorial in TutorialExplorer.
+ */
+
+require('babel-polyfill');
+import React from 'react';
+import shapes from './shapes';
+import getTagString from './util';
+
+const styles = {
+  tutorialDetailModalHeader: {
+    borderBottomWidth: 0,
+    paddingTop: 0,
+    paddingBottom: 4,
+    height: 48
+  },
+  tutorialDetailModalBody: {
+    paddingTop: 0,
+    overflow: 'hidden',
+    textAlign: 'left'
+  },
+  popupFullWidth: {
+    position: 'absolute',
+    left: 0,
+    top: 0,
+    width: '100%'
+  },
+  tutorialDetailName: {
+    fontFamily: '"Gotham 5r", sans-serif',
+    fontSize: 22
+  },
+  tutorialDetailSub: {
+    fontFamily: '"Gotham 3r", sans-serif',
+    fontSize: 12,
+    paddingBottom: 20
+  },
+  tutorialDetailDescription: {
+    fontFamily: '"Gotham 3r", sans-serif',
+    fontSize: 14
+  },
+  tutorialDetailsTable: {
+    marginTop: 20,
+    width: '100%'
+  },
+  tutorialDetailsTableTitle: {
+    padding: 5,
+    width: '40%',
+    fontFamily: '"Gotham 5r", sans-serif',
+    border: '1px solid lightgrey'
+  },
+  tutorialDetailsTableBody: {
+    padding: 5,
+    border: '1px solid lightgrey'
+  },
+};
+
+const TutorialDetail = React.createClass({
+  propTypes: {
+    showing: React.PropTypes.bool.isRequired,
+    item: shapes.tutorial.isRequired,
+    closeClicked: React.PropTypes.func.isRequired,
+  },
+
+  render: function () {
+    if (!this.props.showing) {
+      // Disable body scrolling.
+      $('body').css('overflow', 'auto');
+      return null;
+    }
+
+    // Enable body scrolling.
+    $('body').css('overflow', 'hidden');
+
+    const tableEntries = [
+      {key: 0, title: "Length",                  body: getTagString("length", this.props.item.tags_length)},
+      {key: 1, title: "Subjects",                body: getTagString("subject", this.props.item.tags_subject)},
+      {key: 2, title: "Educator Experience",     body: getTagString("teacher_experience", this.props.item.tags_teacher_experience)},
+      {key: 3, title: "Student Experience",      body: getTagString("student_experience", this.props.item.tags_student_experience)},
+      {key: 4, title: "Type of Activity",        body: getTagString("activity_type", this.props.item.tags_activity_type)},
+      {key: 5, title: "International Languages", body: getTagString("international_languages", this.props.item.tags_international_languages)},
+    ];
+
+    return (
+      <div
+        id="tutorialPopupFullWidth"
+        style={styles.popupFullWidth}
+      >
+        <div
```

Comment:
Would like it if at some point we split out the dialog logic from this into a common PegasusDialog component, and then just have the body portion be part of this component.